e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/type-of-policy.spec.js

```diff
@@ -0,0 +1,267 @@
+import {
+  headingCaption,
+  heading,
+  inlineErrorMessage,
+  submitButton,
+  saveAndBackButton,
+} from '../../../pages/shared';
+import { insurance } from '../../../pages';
+import partials from '../../../partials';
+import {
+  BUTTONS,
+  ERROR_MESSAGES,
+  LINKS,
+  ORGANISATION,
+  PAGES,
+  TASKS,
+} from '../../../../../content-strings';
+import { FIELDS } from '../../../../../content-strings/fields/insurance/policy-and-exports';
+import { ROUTES, FIELD_IDS } from '../../../../../constants';
+import getReferenceNumber from '../../../helpers/get-reference-number';
+
+const { taskList } = partials.insurancePartials;
+
+const CONTENT_STRINGS = PAGES.INSURANCE.POLICY_AND_EXPORTS.TYPE_OF_POLICY;
+
+const FIELD_ID = FIELD_IDS.INSURANCE.POLICY_AND_EXPORTS.POLICY_TYPE;
+
+const singlePolicyField = insurance.policyAndExport.typeOfPolicy[FIELD_ID].single;
+const multiplePolicyField = insurance.policyAndExport.typeOfPolicy[FIELD_ID].multi;
+
+const goToPageDirectly = (referenceNumber) => {
+  cy.visit(`${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.POLICY_AND_EXPORTS.TYPE_OF_POLICY}`, {
+    auth: {
+      username: Cypress.config('basicAuthKey'),
+      password: Cypress.config('basicAuthSecret'),
+    },
+  });
+};
+
+context('Insurance - Policy and exports - Type of policy page - As an exporter, I want to enter the type of policy I need for my export contract', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.visit(ROUTES.INSURANCE.START, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    taskList.prepareApplication.tasks.policyTypeAndExports.link().click();
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      const expected = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.POLICY_AND_EXPORTS.TYPE_OF_POLICY}`;
+      cy.url().should('eq', expected);
+    });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  it('passes the audits', () => {
+    cy.lighthouse({
+      accessibility: 100,
+      performance: 75,
+      'best-practices': 100,
+      seo: 70,
+    });
+  });
+
+  it('renders a back link with correct url', () => {
+    partials.backLink().should('exist');
+    partials.backLink().invoke('text').then((text) => {
+      expect(text.trim()).equal(LINKS.BACK);
+    });
+
+    partials.backLink().click();
+
+    const expectedUrl = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.ALL_SECTIONS}`;
+
+    cy.url().should('include', expectedUrl);
+
+    goToPageDirectly(referenceNumber);
+  });
+
```

Comment:
once my header ticket is in, we will need to add a quick test to make sure the header href goes to the right place

Reply:
Can't seem to find this in companies house tests, where is this test please?

Reply:
will need main-application merge and to add
`it('should render a header with href to insurance start', () => {
    partials.header.serviceName().should('have.attr', 'href', insuranceStartRoute);
  });`

Reply:
Thanks, done 😊